inc/httpRequest.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef HTTPREQUEST_HPP
+#define HTTPREQUEST_HPP
+
+#include <iostream>
+#include <map>
+#include <string.h>
+#include <string>
+#include <vector>
+
+class HttpRequest
+{
+  public:
+	typedef std::map<std::string, std::string> t_object;
+	t_object								   _request_map;
+
+	HttpRequest();
+	~HttpRequest();
+
+	void parseBuffer(char *buff);
```

Comment:
Can we set this pointer read-only ?
```suggestion
	void parseBuffer(const char *buff);
```